modin/pandas/test/dataframe/test_udf.py

```diff
@@ -144,6 +153,11 @@ def apply_func(series, y):
     )
 
 
+@pytest.mark.xfail(
```

Comment:
this bug became visible because of the [changes of handling DataFrames](https://github.com/pandas-dev/pandas/issues/42403) at the `DataFrame.insert`

Reply:
@dchigarev, will you look at it after this pandas update?

Reply:
ok